src/WrappedVault.sol

```diff
@@ -351,17 +353,17 @@ contract WrappedVault is Owned, ERC20, IWrappedVault {
 
         // If there are no stakers we just change the last update time, the rewards for intervals without stakers are not accumulated
 
-        uint256 elapsedWAD = elapsed * 1e18;
+        // The rewards per token are scaled up for precision
+        uint256 elapsedWAD = elapsed * RPT_PRECISION;
```

Comment:
A little misleading to call this variable `elapsedWAD`. Worth renaming to `elapsedScaled` for readability

Reply:
Changed! ty